### DISCUSSION TITLE
How to save solution and plot solution at every iteration

### DISCUSSION THREAD

**Discussion by salaudeen-ya** [idaholab/moose#23320] [Q&A General] [answered]
Hello,

I am trying to solve a simple laplace equation in 2D through MultiAPP coupling. I have a working input file but I would like to save the solution at every iteration and then be able to plot it. How best to do that?

------------
Paraview Question
--------------

I am trying to plot two different systems beside each other. The variation of the temperature in these systems should vary accordingly. However, Paraview is given me different color map and I need these two images (attached) to look exactly like the single image(also attached). How do I do that?

ND: The magnitude in the two images are exactly the same. Only that the color representation in the first two images made it look like the image on the right has zero values.

![image](https://user-images.githubusercontent.com/38023107/216396904-a63c9609-3a2f-43ed-85a1-221ae0c1b391.png)

![image](https://user-images.githubusercontent.com/38023107/216397159-42402eef-272b-46dd-a0ee-77af721e7a53.png)

**Comment by GiudGiud** [comment, marked as answer]
In Paraview you can use the colobar scaling with the "C" on it (for custom) to set the same manual bounds for both visualizations

**Reply by GiudGiud** [reply]
Oh I missed the main question

You can output on linear and nonlinear iterations by setting the execute_on parameter of the Outputs block (or the exodus subblock of the [Ouputs]).
So something like: `execute_on = 'linear nonlinear timestep_end'`

Guillaume
